cmd/kyma/provision/k3d/cmd.go

```diff
@@ -22,27 +22,27 @@ type command struct {
 
 //NewCmd creates a new k3d command
 func NewCmd(o *Options) *cobra.Command {
-
 	c := command{
 		Command: cli.Command{Options: o.Options},
 		opts:    o,
 	}
 
 	cmd := &cobra.Command{
 		Use:     "k3d",
-		Short:   "Provisions a Kubernetes cluster based on k3d v4.",
+		Short:   "Provisions a Kubernetes cluster based on k3d v5.",
 		Long:    `Use this command to provision a k3d-based Kubernetes cluster for Kyma installation.`,
 		RunE:    func(_ *cobra.Command, _ []string) error { return c.Run() },
 		Aliases: []string{"k"},
 	}
 
 	cmd.Flags().StringVar(&o.Name, "name", "kyma", `Name of the Kyma cluster`)
 	cmd.Flags().IntVar(&o.Workers, "workers", 1, "Number of worker nodes (k3d agents)")
-	cmd.Flags().StringSliceVarP(&o.ServerArgs, "server-arg", "s", []string{}, "One or more arguments passed to the Kubernetes API server (e.g. --server-arg='--alsologtostderr')")
-	cmd.Flags().StringSliceVarP(&o.AgentArgs, "agent-arg", "a", []string{}, "One or more arguments passed to the k3d agent command on agent nodes (e.g. --agent-arg='--alsologtostderr')")
+	cmd.Flags().StringSliceVarP(&o.K3sArgs, "k3s-arg", "s", []string{}, "One or more arguments passed from k3d to the k3s command (format: ARG@NODEFILTER[;@NODEFILTER])")
 	cmd.Flags().DurationVar(&o.Timeout, "timeout", 5*time.Minute, `Maximum time for the provisioning. If you want no timeout, enter "0".`)
 	cmd.Flags().StringSliceVarP(&o.K3dArgs, "k3d-arg", "", []string{}, "One or more arguments passed to the k3d provisioning command (e.g. --k3d-arg='--no-rollback')")
 	cmd.Flags().StringVarP(&o.KubernetesVersion, "kube-version", "k", "1.20.11", "Kubernetes version of the cluster")
+	cmd.Flags().StringSliceVar(&o.UseRegistry, "registry-use", []string{}, "Connect to one or more k3d-managed registries. Kyma automatically creates a registry for serverless images.")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&o.UseRegistry, "registry-use", []string{}, "Connect to one or more k3d-managed registries. Kyma automatically creates a registry for Serverless images.")
```
Unless I got it wrong and it's not what we mean here.